recipes/boost/all/conanfile.py

```diff
@@ -760,6 +760,18 @@ def create_library_config(name):
         filename = "%s/user-config.jam" % folder
         tools.save(filename,  contents)
 
+    @property
+    def _toolset_version(self):
+        if self._is_msvc:
+            compiler_version = str(self.settings.compiler.version)
+            if Version(compiler_version) >= "16":
+                return "14.2"
+            elif Version(compiler_version) >= "15":
+                return "14.1"
```

Comment:
Should this match the actual toolset in the Conan profile?

Reply:
to be honest, no idea (yet). that's the same like it was in previous version (so far, nobody complained)